core/merginapi.cpp

```diff
@@ -3833,6 +3835,19 @@ void MerginApi::getServerConfigReplyFinished()
         }
       }
 
+      if ( ( major >= 2024 && minor >= 4 ) && diagnosticUrl.isEmpty() )
```

Comment:
I think it is since `2025` :)
```suggestion
      if ( ( major >= 2025 && minor >= 4 ) && diagnosticUrl.isEmpty() )
```